packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewModalContent.tsx

```diff
@@ -232,6 +233,7 @@ export const TransactionReviewModalContent = ({
                         outputs={outputs}
                         buttonRequestsCount={buttonRequestsCount}
                         isRbfAction={isRbfAction}
+                        isTradingAction={!!isTradingAction}
```

Comment:
I think it should be boolean in the const already according its name. Could you put `!!` there if it's really necessary? I guess it's because `undefined`?